packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -46,26 +49,33 @@ export const SideNavListItemContainer = styled("li", {
   "&:last-child": {
     marginBottom: odysseyDesignTokens.Spacing2,
   },
-  "& a": {
+  "& a, & div[role='button']": {
```

Comment:
I think there's a better way of doing this than targeting all buttons and links. The way this is written, it's very flakey.

It's possible to check the parent component in the child component such as `[data-collapsed] & { ... }`.

Reply:
refactored